p2p_wallet/Scenes/Main/BankTransfer/StrigaRegistration/SecondStep/StrigaRegistrationSecondStepViewModel.swift

```diff
@@ -18,8 +18,8 @@ final class StrigaRegistrationSecondStepViewModel: BaseViewModel, ObservableObje
     @SwiftyUserDefault(keyPath: \.strigaOTPResendCounter, options: .cached)
     private var resendCounter: ResendCounter?
 
-    @SwiftyUserDefault(keyPath: \.bankTransferLastCountry, options: .cached)
-    private var lastChosenCountry: Country?
+//    @SwiftyUserDefault(keyPath: \.bankTransferLastCountry, options: .cached)
+//    private var lastChosenCountry: Country?
```

Comment:
any problems here? should be region instead?

Reply:
Registration shouldn't override Apps country